test-app/tsconfig.json

```diff
@@ -1,6 +1,13 @@
 {
   "extends": "@tsconfig/ember/tsconfig.json",
   "compilerOptions": {
+    /**
+      https://www.typescriptlang.org/tsconfig#verbatimModuleSyntax
+
+      We don't want to include types dependencies in our compiled output, so tell TypeScript
+      to enforce using `import type` instead of `import` for Types.
+     */
+    "verbatimModuleSyntax": true,
```

Comment:
It is a part of the base config https://github.com/tsconfig/bases/blob/8fee7129ada13c7c779472ebb29130fa9581cf1c/bases/ember.json#L17
Why do we need to specify it here?

Reply:
oh depends on which version you have installed, I forgot to check that -- its fairly recent

Reply:
I've updated it to the latest in the original PR

Reply:
Excellent!